runtime/autoload/remote/host.vim

```diff
@@ -118,11 +123,20 @@ endfunction
 
 function! remote#host#LoadRemotePlugins() abort
   if filereadable(s:remote_plugins_manifest)
+        \ && !s:loaded_remote_plugins
     exe 'source '.s:remote_plugins_manifest
+    let s:loaded_remote_plugins = 1
```

Comment:
Why is this flag needed? `empty(s:hosts)` is checked by `remote#host#Require()`.